link_grammar_web_ui/settings.py

```diff
@@ -11,10 +11,17 @@
 
 MANAGERS = ADMINS
 
-DATABASES = {}
-# Parse database configuration from $DATABASE_URL
-import dj_database_url
-DATABASES['default'] = dj_database_url.parse('postgres://iciulirvocekem:5ZSvHQFuoUOU4313Wgml3sZ3UI@ec2-54-235-154-147.compute-1.amazonaws.com:5432/d5p7hf0qnvvpeu')
+#Configurations for Brown database in Heroku
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'd8fql8pc0r1fv2',
+        'USER': 'hlxuersbxzffyf',
+        'PASSWORD': 'DS59cTgRW7bfjowvfBQBTsi2Vs',
```

Comment:
It seems concerning that we have db credentials present in a publicly visible repository...